### ISSUE TITLE
[alexnet] what about the alexnet result

### ISSUE THREAD

**Issue by qiaohaijun** [dmlc/mxnet#758]
I try inception-bn, the result is almost the same with the one that guide show, but when I try alexnet I failed. The validation-accuracy of alexnet model is very low.

**Comment by qiaohaijun** [comment]
7 x 4 k40m
use the surperparameter of googlenet with bn

**Comment by antinucleon** [comment]
Batch Norm enable us train with much larger learning rate. However, large learning rate is harmful for many other network without batch norm.

**Comment by qiaohaijun** [comment]
I change lr to 0.005 and wd to 0.0005 get better result

**Comment by qiaohaijun** [comment]
@antinucleon 
I use lr =0.05 and wd =0.005 this configure work fine on caffe alexnet model, but not well in mxnet. Can you give your configure?

**Comment by antinucleon** [comment]
It may be caused by different gradient scale method. Usually without BN, I use LR=1e-2 at start.

**Comment by qiaohaijun** [comment]
@antinucleon I try again

**Comment by qiaohaijun** [comment]
2015-12-01 20:25:35,285 Node[0] Epoch[0] Train-accuracy=0.107807
2015-12-01 20:25:35,285 Node[0] Epoch[0] Time cost=1296.379
2015-12-01 20:25:53,233 Node[0] Epoch[0] Validation-accuracy=0.204680

**Comment by qiaohaijun** [comment]
but if use the original alexnet model, I get the following result:

2015-12-01 20:46:17,753 Node[0] Update[2503]: Change learning rate to 0.04700
2015-12-01 20:46:18,138 Node[0] Epoch[0] Train-accuracy=0.000969
2015-12-01 20:46:18,138 Node[0] Epoch[0] Time cost=1006.859
2015-12-01 20:46:39,549 Node[0] Epoch[0] Validation-accuracy=0.001156

**Comment by qiaohaijun** [comment]
one machine four gpu cards
k40m

**Comment by qiaohaijun** [comment]
@antinucleon I change the initialization parameter by hard code, and then a get a result that similar with caffe result.

**Comment by anijain2305** [comment]
@qiaohaijun Can you please share the hard coded parameters which helped you get high accuracy?
